ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-ref.yaml

```diff
@@ -31,11 +28,11 @@ ref:
       - local-zone
       - wavelength-zone
       - local-zone,wavelength-zone
```

Comment:
@yunjiang29 noticed it's also possible to set `local-zone,wavelength-zone` for `EDGE_ZONE_TYPES`, do we support a cluster to be installed with nodes from both local-zone and wavelength-zone in the same time?

Reply:
@gpei  Technically, yes, but it's not a typical user scenario,  [here](https://github.com/openshift/release/pull/47045#issuecomment-1882397382) is the reason of introducing `EDGE_ZONE_TYPES`